### DISCUSSION TITLE
IIS deployment

### DISCUSSION THREAD

**Discussion by mabdalla123** [abpframework/abp#18852] [Q&A]
i am trying to deploy the app to local iis server , i can't quite figure it the documentation is not clear about this or i just cant understand it 
in my application i dont have a sperate auth server and in the documentation there is , iam i messing a file or project ?

can any one write a tutorial on how to deploy mvc not tierd application or direct me to a resource 
thank you

**Comment by maliming** [comment]
hi

> Deploying an ABP application is not different than deploying any .NET or ASP.NET Core application. However, there are some topics that you should care about when you are deploying your applications. You can check ABP's [Deployment documentation](https://docs.abp.io/en/abp/latest/Deployment/Index) before deploying your application.
